content/docs/apps/continuous-deployment.md

```diff
@@ -51,7 +53,7 @@ $ cf delete-service my-deployer-account
 
 ### *East/West environment:* Ask an admin
 
-The first thing you need to do is ask an admin to setup a "deployer" user in your organization and give it permission to deploy to the desired space:
+Ask an admin to setup a "deployer" user in your organization and give it permission to deploy to the desired space:
```

Comment:
Oh this is a good opportunity to address https://github.com/18F/cg-site/issues/453 - maybe just "Ask cloud.gov support to set up a "deployer" user in your organization and give it permission to deploy to the desired space." and delete the commands? (Presumably we know the command to use or have it documented elsewhere.)